docs/designers-developers/developers/block-api/block-supports.md

```diff
@@ -163,3 +309,40 @@ supports: {
     reusable: false
 }
 ```
+
+## spacing
```

Comment:
Is this stable? I mean the block support is stable but I'm not sure we have a stable way to enable the UI for themes.

Reply:
It looks like it is, as per https://github.com/WordPress/gutenberg/pull/25788

Reply:
Added a comment about the things that require theme support to work https://github.com/WordPress/gutenberg/pull/26771/commits/67dd1c5285cb41ab3a95163c7e183f21b9445ecb

Reply:
Prepared https://github.com/WordPress/gutenberg/pull/26859 to tweak the docs and signal that `custom-spacing` theme support still requires the plugin to work.